.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: Node.js CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Should probably enable pull requests on non-master branch (i.e, feature branch), too.